Makefile

```diff
@@ -30,6 +30,7 @@ godeps:
 	go get -u github.com/miekg/dns
 	go get -u github.com/prometheus/client_golang/prometheus/promhttp
 	go get -u github.com/prometheus/client_golang/prometheus
+	go get -u github.com/Knetic/govaluate
```

Comment:
why can't this be vendored?

Reply:
It can be. I just do not know how to do it.